### PR TITLE
PC-47 Doc moving mock servers

### PR DESCRIPTION
**Pull request by jkonrath-postman**
* Moved contents to top above getting started
* Added section on moving mock servers + image

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/mocking-data/setting-up-mock.md [hunk 1]
```diff
@@ -212,6 +214,14 @@ If you see `No matching requests` listed in the __Response__ column, this may me
 
 In the case of a service outage, you will get a 502/503/504 response. Please subscribe and check the Postman [status page](https://status.postman.com/) for updates if you encounter this.
 
+## Moving mock servers
+
+After you've created a mock server, you can move it to another workspace. This enables you to collaborate seamlessly with other team members working on the APIs and will also improve the workflow of API development.
+
+To move a mock server, click the **View more actions** menu on the Mock server, and click **Move**. You will be asked to select a workspace. Click **Move and Contine** to move the mock server.
```
Comment (sowmyaraj92) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/setting-up-mock.md:221]: I would suggest rephrasing 
"click the **View more actions** menu on the Mock server, and click **Move**."  > 
"click **View more actions (...)** on the Mock server, and select **Move**.

Comment (sowmyaraj92) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/setting-up-mock.md:221]: Typo fix - **Move and Continue**

File: src/pages/docs/designing-and-developing-your-api/mocking-data/setting-up-mock.md [hunk 2]
```diff
@@ -212,6 +214,14 @@ If you see `No matching requests` listed in the __Response__ column, this may me
 
 In the case of a service outage, you will get a 502/503/504 response. Please subscribe and check the Postman [status page](https://status.postman.com/) for updates if you encounter this.
 
+## Moving mock servers
+
+After you've created a mock server, you can move it to another workspace. This enables you to collaborate seamlessly with other team members working on the APIs and will also improve the workflow of API development.
```
Comment (sowmyaraj92) [inline src/pages/docs/designing-and-developing-your-api/mocking-data/setting-up-mock.md:219]: Just a suggestion - "After you've created a mock server, you can move it to another workspace. This enables you to collaborate seamlessly with other team members working on the APIs ~and will also~ to improve the API development workflow."

**Review by sowmyaraj92 — COMMENTED** [review]
Looks good, just minor changes! 👍

**Review by vvs11 — APPROVED** [review]
Looks good

**Review by sowmyaraj92 — APPROVED** [review]
Looks good 🎉

**Pull request merged**